deploy_packages.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import sys
+import os
+import shutil
+from shutil import copyfile
+
+deployable_packages = ['common', 'datamodel', 'parsers', 'gl_extensions', 'input_manager', 'renderer', 'ticker', 'traits', 'window_manager', 'camera'] # left for later use
+
+def usage():
```

Comment:
Add a TODO to make a proper program options parse, with proper description and usage.